README.md

```diff
@@ -107,34 +107,28 @@ $ make watch channel=my-dev-channel
 info gazer-color Watching 1 file[s] (replicated.yaml)
 ```
 
-#### Run CLI in Docker
+#### CLI in Docker
 
-Use the `Dockerfile` to execute the Replicated CLI inside a container. This is especially helpful if you are running on Windows since `make` and `replicated` cli are not supported. If you have OS X or Linux you can continue using `make`.
+Use `replicated/vendor-cli` Docker image to execute the CLI inside a container. This is useful in environments where `make` and `replicated` cli are unsupported. If you have OS X or Linux you can continue using `make`.
 
-Build the image
-```
-docker build -t replicated-cli .
-```
 
-Run Replicated CLI container to verify
+You can either build the image from scratch or...
 ```
-docker run -t replicated-cli --help
+docker build -t replicated/vendor-cli .
 ```
 
-If the above command was successful you can run the following `make` commands. If `make` is not available just copy the respective commands from `Makefile` to execute separately.
-
-Note: Complete the 'Configure environment' section as the following commands require those environment variables.
-
-List releases
+List releases which will download `replicated/vendor-cli` and verify its copacetic.
```

Comment:
Can we add a little more detail here around what is copacetic? I think the initial intent for `make list-releases` was to ensure that the env vars were configured correctly.